buildpack.md

```diff
@@ -91,6 +80,54 @@ Buildpack API versions:
  - `<major>` and `<minor>` MUST only contain numbers (unsigned 64 bit integer)
  - When `<major>` is greater than `0` increments to `<minor>` SHALL exclusively indicate additive changes
 
+## Terminology
+
+### CNB Terminology
```

Comment:
Feedback from 4/21 Working Group: (if possible, without making each term its own section) we should link back to this section as much as is practical.